<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLocation</class>
 <widget class="QDialog" name="DialogLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>列  号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxColumn_2">
        <property name="minimum">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkBoxColumn_2">
        <property name="text">
         <string>列增</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>行  号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chkBoxRow_2">
        <property name="text">
         <string>行增</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxRow_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>设定文字</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="edtCaption_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSetText_2">
       <property name="text">
        <string>设定文字</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Course/chap06Forms/samp6_2CustomDialogs/res.qrc">
         <normaloff>:/images/images/506.bmp</normaloff>:/images/images/506.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose_2">
       <property name="text">
        <string>关  闭</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Course/chap06Forms/samp6_2CustomDialogs/res.qrc">
         <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Course/chap06Forms/samp6_2CustomDialogs/res.qrc"/>
 </resources>
 <connections/>
</ui>
